modules/acquia_telemetry/src/Telemetry.php

```diff
@@ -0,0 +1,205 @@
+<?php
+
+namespace Drupal\acquia_telemetry;
+
+use Drupal\Component\Serialization\Json;
+use Drupal\Component\Utility\Crypt;
+use Drupal\Component\Utility\NestedArray;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Extension\ModuleExtensionList;
+use GuzzleHttp\ClientInterface;
+
+/**
+ * Telemetry service.
+ */
+class Telemetry {
+
+  /**
+   * Amplitude API URL.
+   *
+   * @see https://developers.amplitude.com/#http-api
+   */
+  const AMPLITUDE_API_URL = 'https://api.amplitude.com/httpapi';
```

Comment:
I'm not a huge fan of these constants for two reasons:
1) They're exposed to outside code. It's not possible to make them private until PHP 7.1, which we don't yet support, sadly.
2) They're basically impossible to override for testing. This is the reason I wanted them as simple config values; if they were, we could unit test this class, and simulate HTTP requests, by passing in different config values. That said, this is a lesser concern because we _can_ mock the HTTP client and simulate the HTTP requests that way.

I'm not sure we _really_ need to change these, seeing as how we're not exactly designing a long-term API; just calling it out. One compromise, to fix the visibility issue, is for these to be `private static` properties.